zhaquirks/sinope/__init__.py

```diff
@@ -23,7 +23,7 @@
 
 SINOPE = "Sinope Technologies"
 SINOPE_MANUFACTURER_CLUSTER_ID = 0xFF01
-ATTRIBUTE_ACTION = "actionReport"
+ATTRIBUTE_ACTION = "action_report"
```

Comment:
Changing this might cause issues with current automations that do not use device triggers, right?